src/Home/Weather/Weather.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 
 function Weather() {
-    return <div>
-        <iframe className='weatherTable' src='https://mars.nasa.gov/layout/embed/image/mslweather/' width='100%' height='650'  scrolling='no'></iframe>
+    return <div className='weatherContainer'>
+        <iframe className='weatherTable' src='https://mars.nasa.gov/layout/embed/image/mslweather/' width='100%' height='600'  scrolling='no' frameborder='0'></iframe>
```

Comment:
this frameBorder=0 isn't being compiled very well in the browsers so I'm not sure we want it in

Reply:
We have actioned this :)